tasks/install.yml

```
@@ -71,27 +71,31 @@
 - name: Install | Tailscale Status
   ansible.builtin.debug:
     var: tailscale_status
-  when: verbose | bool
+  when: verbose
 
 - name: Install | Record Tailscale Version
   ansible.builtin.command: tailscale version
   changed_when: false
   register: tailscale_version
-  when: verbose | bool
+  when: verbose
 
 - name: Install | Tailscale Version
   ansible.builtin.debug:
     msg: "{{ tailscale_version.stdout.split('\n') }}"
-  when: verbose | bool
+  when: verbose
 
-- name: Install | Calculate Changes to Tailscale
-  ansible.builtin.include_tasks: lib/up-state.yml
-  loop:
-    - "pull"
+- name: Install | Write state file
```

Nitpick, but suggested rename of this step:

```suggestion
- name: Install | Save State
```